### PR TITLE
changed default divider color

### LINKED ISSUE #6272: [Bug] Default color for divider widget is bad, should be a softer grey
## Description

The default color used on the divider widget is heavy black that doesn't match with the UI, it should a softer grey like #C4C4C4 so the widget can be used without any config.

### Steps to reproduce the behaviour:

Drop divider widget onto the canvas, the default color is #000000

### Reference Images

Image from iOS
Image from iOS (1)

### PR DESCRIPTION
**Pull request by techbhavin**
## Description

> Default color for divider widget is bad, should be a softer grey

Fixes #6272 

## Type of change
- Bugfix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ]  I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ]  I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

File: app/client/src/mockResponses/WidgetConfigResponse.tsx [hunk 1]
```diff
@@ -1159,7 +1159,7 @@ const WidgetConfigResponse: WidgetConfigReducerState = {
       capType: "nc",
       capSide: 0,
       strokeStyle: "solid",
-      dividerColor: "black",
+      dividerColor: "#C4C4C4",
```
Comment (somangshu) [inline app/client/src/mockResponses/WidgetConfigResponse.tsx:1162]: Can we reference this from the theme?

**Review by somangshu — CHANGES_REQUESTED** [review]
Changes as mentioned

File: app/client/src/mockResponses/WidgetConfigResponse.tsx (hunk 1 above)
Comment (techbhavin) [reply app/client/src/mockResponses/WidgetConfigResponse.tsx:1162]: @somangshu  here this color is not present in the theme.

Comment (somangshu) [reply app/client/src/mockResponses/WidgetConfigResponse.tsx:1162]: Please add it in the theme grey scale then

File: app/client/src/constants/Colors.tsx [hunk 2]
```diff
@@ -77,6 +77,7 @@ export const Colors: Record<string, string> = {
   NERO: "#2B2B2B",
   CHARCOAL: "#4B4848",
   LIGHT_GREY: "#D4D4D4",
+  GREY_SCALE: "#C4C4C4",
```
Comment (somangshu) [inline app/client/src/constants/Colors.tsx:80]: Please use LIGHT_GREY2 for the variable name

**Review by somangshu — DISMISSED** [review]
Approving for tests to run

**Review by somangshu — APPROVED** [review]
LGTM

**Pull request merged**